test/common.py

```diff
@@ -2335,6 +2370,15 @@ def browser_restart(cls):
       logger.info('Browser did not respond to `terminate`.  Using `kill`')
       cls.browser_proc.kill()
       cls.browser_proc.wait()
+    if cls.browser_data_dir:
+      utils.delete_dir(cls.browser_data_dir)
+      cls.browser_data_dir = None
```

Comment:
It might be useful for debugging to leave this around after terminate, and instead clean it up on startup.